microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -1298,3 +1304,47 @@ def verify_waagent_version(self, node: Node) -> None:
                     "extensions/update-linux-agent?tabs=ubuntu for more details to "
                     "update."
                 )
+
+    @TestCaseMetadata(
+        description="""
+        This test verifies the version of Python installed on the system.
+
+        Steps:
+        1. Retrieve the Python version.
+        2. Check if the version is lower than the minimum supported version.
+           The minimum supported version can be found at:
+           https://devguide.python.org/versions/
+        3. Fail the test if the version is lower than the minimum supported version,
+           otherwise pass.
+        """,
+        priority=1,
+        requirement=simple_requirement(supported_platform_type=[AZURE, READY, HYPERV]),
+    )
+    def verify_python_version(self, node: Node) -> None:
+        minimum_version = Version("3.8.0")
+        python_commands = ["python3 --version", "python --version"]
```

Comment:
It should always support python3, if it's Linux. So it doesn't need to check `python`.

Reply:
Fixed